### PR TITLE
Feat/bs 13259 add maven submodule for custompages overview

### PR DESCRIPTION
**Pull request by nchabanoles**
Setup the Case Overview page project (as it was on the last sprint, still more work needed to complete it).
The page will not be available in the bundle yet. Wait for the next PR on the distrib project for that...

### CODE REVIEW

File: bonita-home/src/main/resources/client/platform/work/i18n/portal-js.po [hunk 1]
```diff
@@ -17,273 +17,1092 @@ msgstr ""
...
+msgstr ""
+
+#: main/features/admin/processes/details/process-managers-mapping.html
+#: main/features/admin/processes/details/actors-mapping.html
+msgid "To do so, go to"
+msgstr ""
+
+msgid ""
+"To use this feature, create a new version of the process in Bonita BPM "
+"Studio version 6.4 and above, then create and install the new .bar file in "
```
Comment (apupier) [inline bonita-home/src/main/resources/client/platform/work/i18n/portal-js.po]: Bonita BPM Studio should externalized in order to be able to override it for OEM

File: custom-pages/case-autogenerated-overview/.jshintrc [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    "expect": false,
+    "inject": false,
+    "it": false,
+    "jasmine": false,
+    "require": false,
+    "spyOn": false,
+    "protractor":false,
+    "element":false
+  }
+}
```
Comment (apupier) [inline custom-pages/case-autogenerated-overview/.jshintrc]: we have a jshintrc for every module? not possible to reference one in a single place?

File: custom-pages/case-autogenerated-overview/gulpfile.js [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+
+function serve(configuration) {
+  return connect.server({
+    root: ['src', '.'],
+    port: opt.port,
+    livereload: configuration.livereload,
+    middleware: function () {
+      return [
+        rewriter([
+          '^/bonita/portal http://localhost:8080/bonita/portal [P]',
```
Comment (apupier) [inline custom-pages/case-autogenerated-overview/gulpfile.js]: 8080 hardcoded? What is happening when the port has been updated in the Studio?

File: custom-pages/case-autogenerated-overview/package.json [hunk 4]
```diff
@@ -0,0 +1,51 @@
+{
+  "name": "bonita-custom-page",
+  "version": "1.0.0",
```
Comment (apupier) [inline custom-pages/case-autogenerated-overview/package.json]: we start them in 1.0.0 or 7.0.0?

File: custom-pages/case-autogenerated-overview/pom.xml [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+								<nodeVersion>v0.10.33</nodeVersion>
+								<npmVersion>2.1.7</npmVersion>
+							</configuration>
+						</execution>
+					</executions>
+				</plugin>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-compiler-plugin</artifactId>
+					<version>2.5.1</version>
```
Comment (apupier) [inline custom-pages/case-autogenerated-overview/pom.xml]: we are overriding the version dfined in bonita-web? <plugin>
                    <groupId>org.apache.maven.plugins</groupId>
                    <artifactId>maven-compiler-plugin</artifactId>
                    <version>2.3.2</version>
                </plugin>
BTW, it seems that th eone defined int the top parent doesn't have the correct parameters, right?

File: custom-pages/case-autogenerated-overview/pom.xml [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+		</pluginManagement>
+
+
+
+
+		<plugins>
+			<plugin>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<!-- 2.8.0-01 and later require maven-compiler-plugin 3.1 or higher -->
+				<version>3.1</version>
```
Comment (apupier) [inline custom-pages/case-autogenerated-overview/pom.xml]: again another version for maven-compiler-version?

File: server/src/main/java/org/bonitasoft/web/rest/server/api/bpm/process/ProcessDefinitionDesignResource.java [hunk 7]
```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (C) 2015 BonitaSoft S.A.
```
Comment (apupier) [inline server/src/main/java/org/bonitasoft/web/rest/server/api/bpm/process/ProcessDefinitionDesignResource.java]: BonitaSoft --> Bonitasoft

File: server/src/main/java/org/bonitasoft/web/rest/server/api/bpm/process/ProcessDefinitionDesignResource.java [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+ * BonitaSoft, 32 rue Gustave Eiffel - 38000 Grenoble
```
Comment (apupier) [inline server/src/main/java/org/bonitasoft/web/rest/server/api/bpm/process/ProcessDefinitionDesignResource.java]: BonitaSoft --> Bonitasoft

File: server/src/main/java/org/bonitasoft/web/rest/server/api/bpm/process/ProcessDefinitionDesignResourceFinder.java [hunk 9]
```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (C) 2015 BonitaSoft S.A.
```
Comment (apupier) [inline server/src/main/java/org/bonitasoft/web/rest/server/api/bpm/process/ProcessDefinitionDesignResourceFinder.java]: BonitaSoft --> Bonitasoft

File: server/src/main/java/org/bonitasoft/web/rest/server/api/bpm/process/ProcessDefinitionDesignResourceFinder.java [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+ * BonitaSoft is a trademark of BonitaSoft SA.
```
Comment (apupier) [inline server/src/main/java/org/bonitasoft/web/rest/server/api/bpm/process/ProcessDefinitionDesignResourceFinder.java]: BonitaSoft --> Bonitasoft

File: server/src/main/java/org/bonitasoft/web/rest/server/api/bpm/process/ProcessDefinitionDesignResourceFinder.java [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+ * This software file is BONITASOFT CONFIDENTIAL. Not For Distribution.
+ * For commercial licensing information, contact:
+ * BonitaSoft, 32 rue Gustave Eiffel – 38000 Grenoble
```
Comment (apupier) [inline server/src/main/java/org/bonitasoft/web/rest/server/api/bpm/process/ProcessDefinitionDesignResourceFinder.java]: BonitaSoft --> Bonitasoft

File: server/src/test/java/org/bonitasoft/web/rest/server/api/bpm/process/ProcessDefinitionDesignResourceTest.java [hunk 12]
```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright (C) 2015 BonitaSoft S.A.
```
Comment (apupier) [inline server/src/test/java/org/bonitasoft/web/rest/server/api/bpm/process/ProcessDefinitionDesignResourceTest.java]: BonitaSoft --> Bonitasoft

File: server/src/test/java/org/bonitasoft/web/rest/server/api/bpm/process/ProcessDefinitionDesignResourceTest.java [hunk 13]
```diff
@@ -0,0 +1,88 @@
...
+ * BonitaSoft, 32 rue Gustave Eiffel - 38000 Grenoble
```
Comment (apupier) [inline server/src/test/java/org/bonitasoft/web/rest/server/api/bpm/process/ProcessDefinitionDesignResourceTest.java]: BonitaSoft --> Bonitasoft

File: custom-pages/case-autogenerated-overview/gulpfile.js (hunk 3 above)
Comment (nchabanoles) [inline custom-pages/case-autogenerated-overview/gulpfile.js]: This is used in local when developping. There is no impact when embedded in the product.

**Pull request merged**